src/Nokia_LCD.cpp

```diff
@@ -197,7 +225,19 @@ bool Nokia_LCD::send(const unsigned char lcd_byte, const bool is_data) {
 
     // Send the byte
     digitalWrite(kCe_pin, LOW);
-    shiftOut(kDin_pin, kClk_pin, MSBFIRST, lcd_byte);
+    if (kUsingHardwareSPI) {
+        SPI.begin();
+        SPI.setClockDivider(SPI_CLOCK_DIV4);
+        SPI.setDataMode(SPI_MODE0);
+        SPI.setBitOrder(MSBFIRST);
+        SPI.transfer(lcd_byte);
+    } else {
+        shiftOut(kDin_pin, kClk_pin, MSBFIRST, lcd_byte);
+    }
+    
+    if (kUsingHardwareSPI) {
+        SPI.end();
+    }
```

Comment:
I guess this `if` statement can also be removed and `SPI.end();` to be moved to the other if statement starting at L228.

Reply:
fixed